tests/src/Common/PerfHarness/PerfHarness.csproj

```diff
@@ -1,12 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\..\dependencies.props" />
```

Comment:
I can try, but I do not think the "new" csproj fully understand the "old" msbuild ways.